tests/profile_init.py

```diff
@@ -0,0 +1,105 @@
+import string
```

Comment:
There is already a profile-module, see https://github.com/IAMconsortium/pyam/tree/main/profile